CHANGELOG-2.1.md

```diff
@@ -0,0 +1,21 @@
+# Changelog since v2.0.0
```

Comment:
Is it worth splitting up which changes are for the sidecar, and which are for the controller?

Reply:
sure

Reply:
Done